plugins/LadspaEffect/swh/CMakeLists.txt

```diff
@@ -13,9 +13,13 @@ ENDIF()
 # Additional compile flags
 if(NOT MSVC)
 	set(COMPILE_FLAGS ${COMPILE_FLAGS} -O3 -c
-		-fomit-frame-pointer -funroll-loops -ffast-math -fno-strict-aliasing
+		-funroll-loops -ffast-math -fno-strict-aliasing
 		${PIC_FLAGS}
 	)
+	# an optimisation flag, but makes it harder to debug
+	if(CMAKE_BUILD_TYPE STREQUAL "Release")
+		set(COMPILE_FLAGS ${COMPILE_FLAGS} -fomit-frame-pointer)
```

Comment:
`-fomit-frame-pointer` is included by default at `-O1` or above, so there's no point adding it manually.

Reply:
So do you mind if i nuke the flag entirely? It's used only for ladspa plugins so would it matter?

Reply:
I got a better idea for now. I've used the gprof debug flag to disable the omit frame pointer flag

Reply:
removed it.